content/blog/hello-world.md

```diff
@@ -1,28 +1,41 @@
+Due to the increasing reliance of scientific research on complex computational pipelines, foundational software in the life sciences has outgrown individual labs and institutes in terms of its scale and impact. This has motivated us to start something more than a code repository for an exciting new project – a new organisation to maintain and support foundational tools in the life sciences in the Python ecosystem. We are therefore very excited to announce **scverse** – a consortium for joint development of major packages in omics data analysis in life sciences.
 
-Our primary experience lies in the field of computational genomics and lately single-cell sequencing technologies, and this has defined our scope and focus in the very beginning of our journey that we have called {{< rawhtml >}}<span class="backticked">scverse</span>{{< /rawhtml >}}.
+## What tools are part of scverse?
 
+Our primary experience lies in the field of computational biology with an emphasis on single-cell omics technologies. This experience has defined our initial scope and focus in the very beginning of our journey. Hence, we start with the two most important data structures for single-cell data in the Python ecosystem, [AnnData](https://github.com/scverse/anndata) for uni-modal data and [MuData](https://github.com/scverse/mudata) for multi-modal data, together with [Scanpy](https://github.com/scverse/scanpy) for general purpose single-cell analysis, [muon](https://github.com/scverse/muon) for multimodal single-cell analysis, [scvi-tools](https://github.com/scverse/scvi-tools) for deep probabilistic analysis of single-cell data and [scirpy](https://github.com/scverse/scirpy) for T-cell receptor analysis. We are looking forward to expanding with more major libraries and frameworks which fit our vision soon.
 
-## What this is about
+## What are we doing?
 
-{{< rawhtml >}}<span class="backticked">scverse</span>{{< /rawhtml >}} is a consortium for joint maintenance of core packages in omics data analysis in life sciences. It has been founded to ensure the long-term maintenance of the core infrastructure in the field.
+Our first priority is to build a strong and lasting community around the tools and the associated analysis. The knowledge to build and maintain our tools should be shared among many to avoid single points of failures. Therefore, we are building tooling to more easily generate scverse based packages and analyses to grow the ecosystem. Furthermore, to bring the community together we are planning workshops and hackathons which will be open to the general community. We are also hoping that a healthy scverse community will naturally guide us to develop tooling where it is the most required.
```

Comment:
```suggestion
Our first priority is to build a strong and lasting community around the core tools powering single cell analysis today.
The knowledge to build and maintain these tools should be shared among many to avoid single points of failures.
Therefore, we are creating open community channels, creating shared infrastructure to ease package development, and building tooling to make creation of scverse based packages easier.
Furthermore, to bring the community together we are planning workshops and hackathons which will be open to the general community.
As we grow, we are hoping to recruit more members into governance roles as well as find even more ways to support and connect our community of developers and users.
```